sources/packages/backend/package.json

```diff
@@ -13,14 +13,14 @@
     "start:dev": "npm run migration:run && nest start --watch --preserveWatchOutput",
     "start:debug": "nest start --debug --watch --preserveWatchOutput",
     "start:prod:db-migrations": "npm run migration:run",
-    "start:prod:api": "node dist/apps/api/main.js",
-    "docker:start:api": "node dist/apps/api/main.js",
-    "start:prod:workers": "node dist/apps/workers/main.js",
-    "docker:start:workers": "node dist/apps/workers/main.js",
-    "start:prod:queue-consumers": "node dist/apps/queue-consumers/main.js",
-    "docker:start:queue-consumers": "node dist/apps/queue-consumers/main.js",
-    "start:prod:load-test-gateway": "node dist/apps/load-test-gateway/main.js",
-    "docker:start:load-test-gateway": "node dist/apps/load-test-gateway/main.js",
+    "start:prod:api": "node --enable-source-maps dist/apps/api/main.js",
+    "docker:start:api": "node --enable-source-maps dist/apps/api/main.js",
+    "start:prod:workers": "node --enable-source-maps dist/apps/workers/main.js",
+    "docker:start:workers": "node --enable-source-maps dist/apps/workers/main.js",
+    "start:prod:queue-consumers": "node --enable-source-maps dist/apps/queue-consumers/main.js",
+    "docker:start:queue-consumers": "node --enable-source-maps dist/apps/queue-consumers/main.js",
+    "start:prod:load-test-gateway": "node --enable-source-maps dist/apps/load-test-gateway/main.js",
+    "docker:start:load-test-gateway": "node --enable-source-maps dist/apps/load-test-gateway/main.js",
```

Comment:
FMU, don't we need to add `source-map-support` as a dependency?

Reply:
NodeJS has already support for that from version 12.12: https://nodejs.org/en/blog/release/v12.12.0/